acceptance/tests/resource/package/common_package_name_in_different_providers.rb

```diff
@@ -14,6 +14,17 @@
   end
 end
 
+step "Verify gem and ruby-devel are installed on agents" do
```

Comment:
I'm thinking this step should be skipped in `aio` since its part of the package, e.g. "gem"